transports/transport-rest/transport-rest-servlet/src/main/java/org/apache/servicecomb/transport/rest/servlet/RestAsyncListener.java

```diff
@@ -63,24 +64,28 @@ public void onTimeout(AsyncEvent event) throws IOException {
     // to avoid concurrent, must lock request
     ServletRequest request = event.getSuppliedRequest();
     HttpServletRequestEx requestEx = (HttpServletRequestEx) request.getAttribute(RestConst.REST_REQUEST);
+    LOGGER.error("Rest request timeout, method {}, path {}.", requestEx.getMethod(), requestEx.getRequestURI());
```

Comment:
maybe we can disable async servlet timeout, just control timeout by threadpool timeout?

Reply:
I disabled it with -1 by default. And I think it's better to leave timeout setting to users if they really need it in some scenario.